code/src/java/pcgen/gui2/facade/CharacterFacadeImpl.java

```diff
@@ -346,14 +346,7 @@ private void initForCharacter()
 		statScoreMap = new HashMap<>();
 		for (PCStat stat : dataSet.getStats())
 		{
-			if (stat instanceof PCStat)
-			{
-				statScoreMap.put(stat, getStatReferenceFacade(stat));
-			}
-			else
-			{
-				statScoreMap.put(stat, new DefaultReferenceFacade<>());
-			}
+			statScoreMap.put(stat, getStatReferenceFacade(stat));
```

Comment:
It is not a logic change.  Part of the reason to remove the interfaces is logic like what used to be here.  StatFacade used to only be implemented by PCStat, so the true part of the IF was the only code it would ever be possible to use.  Eliminating the facade results in no longer needing a cast, and the net result of that means the "stat instanceof PCStat" is universally true, which then clearly identifies the false part of the IF as dead code (Which it was before, but it was non-obvious at the time)

Reply:
I'd add, the compiler is who identified the true part of the IF as always true, so I was taking it's guidance - this is not my intuition making the claim of universally true, it's the compiler's cold mathematics making the statement.